os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -805,6 +805,7 @@ int wifi_connect(
 	}
 
 	result = RTW_SUCCESS;
+	error_flag = RTW_NO_ERROR;

```

Comment:
How about moving it to 795 line after "reason.reason_code = RTK_STATUS_SUCCESS;"

In upper layer, app can check "get info" right after link up.

If error_flag is after link up, there can be timing issue.

Reply:
Updated as suggested.